improvments (#6)
ofirgall/learn-nvim: chapters/03-movements-and-operators.md

* `<` - remove indentation (left)
* `>` - add indentation (right)

> Capital (shift) will make the operation from the cursor to the end of the line, e.g: `D` deletes the line from the cursor to the end of the line. \

Reviewer: ```suggestion
Capital (shift) will apply the operation starting from the cursor to the end of the line, e.g: `D` deletes the line from the cursor to the end of the line. \
```

> Repeating the same operator key will make the operation on the entire line, e.g: `yy` yanks the entire current line.

Reviewer: ```suggestion
Repeating the operator key will apply the operation for the entire line, e.g: `yy` yanks the entire current line.
```
